src/Raven.Studio/typescript/viewmodels/partial/columnsSelector.ts

```diff
@@ -417,7 +417,7 @@ class columnsSelector<T extends object> {
                 
                 if (item.virtualColumn() instanceof hyperlinkColumn) {
                     const href = item.virtualColumn() as hyperlinkColumn<T>;
-                    if (href.header === "Id" && href.valueAccessor.toString().includes("getId()")){
+                    if ((href.header === "Id" || href.header === "@id") && href.valueAccessor.toString().includes("getId()")){
```

Comment:
@ml054 please have a look here

Reply:
this is cherry-picked from @ml054 PR: https://github.com/ravendb/ravendb/pull/18644

Reply:
All good. :)